llama_index/tools/tool_spec/load_and_search/__init__.py

```diff
@@ -0,0 +1 @@
+# init
```

Comment:
nit: might be good to add LoadAndSearchToolSpec to `llama_index/tools/tool_spec/__init__.py` so can be easily imported there

not a big deal though don't want to block